README.md

```diff
@@ -8,9 +8,126 @@ Power Flows DMN - a powerful decision engine.
 
 
 # Getting Started
+Depending on you would like to use the version 2.x.x (recommended as the latest version) or 1.x.x 
+there are 2 different ways of importing dependencies.
+
+## Version 2.x.x
+
+The Power Flows DMN can be imported as a dependency via:
+
+### Maven
+```xml
+<dependency>
+    <groupId>org.powerflows</groupId>
+    <artifactId>dmn-engine</artifactId>
+    <version>2.0.0</version>
+</dependency>
+```
+Since in version 2.0.0 Power Flows DMN has been divided into modules,
+a developer can decide what features have to be imported to his project. 
+Therefore, the first decision will be which modeling format of decision tables to be supported. 
+There is a choice between:
+* YAML file - requires **dmn-io-yaml** dependency
+* XML file - requires **dmn-io-xml** dependency
+* Java/Groovy file (fluent and functional) - delivered with **dmn-engine** dependency
```

Comment:
This suggests to me that dmn-engine delivers groovy functionalities...

Reply:
clarified